lib/client.ts

```diff
@@ -493,9 +484,9 @@ export class Client {
             cancel: async () => {
                 this.logger.info("Stream canceled");
                 this.stateFactory.write();
-                this.strategy.cancel();
-                this.memberManager.close();
-                this.fetcher.close();
+                if (this.strategy) this.strategy.cancel();
+                if (this.memberManager) this.memberManager.close();
+                if (this.fetcher) this.fetcher.close();
```

Comment:
No biggie but I think this can be written a bit more clean like this
```suggestion
                this.strategy?.cancel();
                this.memberManager?.close();
                this.fetcher?.close();
```

Reply:
Fixed in https://github.com/rdf-connect/ldes-client/pull/94/commits/9a30afcb0a541dbee4dddf749a9b85920ac12675